src/js/view/floating/floating-controller.ts

```diff
@@ -220,6 +223,11 @@ export default class FloatingController {
                 transition: null,
                 'transition-timing-function': null
             });
+            setTimeout(() => {
+                this.transitionFloating(false);
+                // Perform resize and trigger "float" event responsively to prevent layout thrashing
+                this._model.trigger('forceResponsiveListener', {});
```

Comment:
This avoids triggering the response listener until the CLS avoiding transition is done.

We could move this into `transitionFloating` with `false`